README.rst

```diff
@@ -84,6 +84,12 @@ Containerized 'Production' environment
     $ invenio-cli containerize --no-install-js
 
 
+Development
+============
+
+To setup the environment for developing on `invenio-cli` itself, check out `this <https://inveniordm.docs.cern.ch/maintenance/newcomers/#tools>`_ paragraph on the InvenioRDM guide for maintainers.
+
```

Comment:
What about a link (in addition or instead) to the `CONTRIBUTING.md` file in this repo? It seems still valid... :thinking:

Reply:
Makes sense, thanks @fenekku !
I'll move the link the `CONTRIBUTING.md` and mention the additional run-tests usage scenario